src/Data/List/Lens.hs

```diff
@@ -89,55 +95,17 @@
 --
 ----------------------------------------------------------------------------
 module Data.List.Lens
-  ( prefixed
-  , suffixed
-  , stripSuffix
+  ( stripSuffix
```

Comment:
`Data.List.Lens` no longer exports any optics at all after this change, which makes the module Haddocks misleading.

Reply:
Alternatively, we could re-export `Prefixed` and `Suffixed` from this module for backwards-compatibility, although that would arguably be inconsistent with how we treat other list-adjacent classes (e.g., `Cons` and `Snoc`).

Reply:
After looking at how many [uses of `Data.List.Lens` there are on Hackage](https://hackage-search.serokell.io/?q=Data%5C.List%5C.Lens), I'm less enthusiastic about simply breaking this outright. For the time being, I've decided to re-export `Prefixed` and `Suffixed` from `Data.List.Lens` to ensure that this change is mostly backwards compatible. I do think we should eventually change this (and I've left a note in the module Haddocks saying as much), but I'll wait until the ecosystem has caught up with `lens-5.1` first to ensure a smoother migration path.